Src/BootCamp.Chapter/Checks.cs

```diff
@@ -9,31 +7,192 @@ public static class Checks
     {
         public static string FindMostPoorPerson(string[] peopleAndBalances)
         {
-            return "";
+            if (Testers.IsThisStringArrayValid(peopleAndBalances))
+            {
+                return "N/A.";
+            }
+            string poorestName = "";
+            decimal amount = 0;
+            int personCount = 1;
+
+            for (int i = 0; i < peopleAndBalances.Length; i++)
+            {
+                var a = new Account(peopleAndBalances[i]);
```

Comment:
The body of the for loop could be refactored into a function (or a few) so that it is easier to skim through the bigger function.

Reply:
i just changed FindPoorestPerson in checks. is this what you meant?